specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/README.md

```diff
@@ -35,13 +45,16 @@ Adding `@usage` and `@access` to `client.tsp`:
 @@access(EventGrid.AcsChatMessageReceivedEventData, Access.public)
 ~~~
 
+
+
 # For Service System Events PR Approval
 
 1) Write your service's system events in TypeSpec.
```

Comment:
nit:  You could save yourself responsibility for explicitly numbering and just use `1)` for each item.   Markdown will automatically number sequentially when it renders.